### PR TITLE
Updated UDP documentation to point to UDP Github README.

### LINKED ISSUE #396: point UDP plugin to new README
https://github.com/influxdb/influxdb/pull/4379 introduced a good [README](https://github.com/influxdb/influxdb/blob/master/services/udp/README.md) for UDP support. Let's link the [UDP section](https://influxdb.com/docs/v0.9/write_protocols/udp.html) to it.

### PR DESCRIPTION
**Pull request by rossmcdonald**
Fixes #396

### CODE REVIEW

File: content/docs/v0.9/write_protocols/udp.md [hunk 1]
```diff
@@ -53,3 +53,7 @@ To write, just send newline separated [line protocol](/docs/v0.9/write_protocols
 ```bash
 $ echo "cpu value=1"> /dev/udp/localhost/8089
 ```
+
+## More Information
+
+For more information about the UDP plugin, please see the UDP plugin README [here](https://github.com/influxdb/influxdb/blob/master/services/udp/README.md).
```
Comment (beckettsean) [inline content/docs/v0.9/write_protocols/udp.md]: We prefer to link meaningful words to the content: 

For more information about the UDP plugin, please see the UDP plugin [README](https://github.com/influxdb/influxdb/blob/master/services/udp/README.md).

**Comment by beckettsean** [comment]
One minor format change. 

Also, for future reference we tend to create PRs directly from within the repo, rather than fork. It makes it easier for us to collaboratively edit if needed.

**Comment by rossmcdonald** [comment]
Update applied. And I'll create a branch in the future, sorry about that!

**Comment by beckettsean** [comment]
:+1: merge it! (we merge our own PRs after another reviewer gives a thumbs up. Some PRs need two, it's a judgement call.)

**Pull request merged**